app/components/progress_tab/monthly_report_component_v2.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+class ProgressTab::MonthlyReportComponentV2 < ApplicationComponent
+  include AssetsHelper
+  include ProgressTabHelper
+
+  MONTH_DATE_FORMAT = Date::DATE_FORMATS[:month_year]
+
+  attr_reader :service, :current_user, :title, :subtitle, :region
+
+  def initialize(service, current_user, title:, subtitle:)
```

Comment:
Some args are named, and other positional. Is this intentional?

Reply:
Will make it all named arguments for consistency